src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -43,7 +43,7 @@ const paddingStyles = css`
     }
 `;
 
-const messageStyles = css`
+const messageStyles = (isThankYouMessage: boolean) => css`
```

Comment:
@oliverlloyd the change to `messageStyles` was only warranted when showing the "Thank You..." message, I've therefore adjusted this so we use the original `headline.medium` if `isThankYouMessage` is `false`.

Reply:
To be honest, I quite liked the smaller version for everything - I think in DCR we have a larger font in general here. But there's an argument to say it's touchable so I think your approach is best